src/components/ModalContent.vue

```diff
@@ -50,7 +50,7 @@
           title="paomedia">www.paomedia.com
         </a>
       </div>
-      <p class="mt-4">Aktuelle Version: 1.8.7</p>
+      <p class="mt-4">Aktuelle Version: 1.1.0</p>
```

Comment:
https://medium.com/hceverything/how-to-show-your-app-version-from-package-json-in-your-vue-application-11e882b97d8c